### ISSUE TITLE
Could not connect to localhost:6969

### ISSUE THREAD

**Issue by m1n1vv** [slooi/NovelAI-Setup-Guide#5]
Installed this morning with no problems. I used it for a few hours and the script stopped working. Is it because of the limit?

```
INFO proxy{id=47ac5018-e3f1-4d9f-b16a-b162caa08c21}: bore_cli::client: new connection
WARN proxy{id=47ac5018-e3f1-4d9f-b16a-b162caa08c21}: bore_cli::client: connection exited with error err=could not connect to localhost:6969
```

**Comment by manaheti1** [comment]
![image](https://user-images.githubusercontent.com/65333593/201042871-6bffe877-89d0-4188-886c-159f1be7fb60.png)
yes if this happened you should change account.

**Comment by m1n1vv** [comment]
> ![изображение](https://user-images.githubusercontent.com/65333593/201042871-6bffe877-89d0-4188-886c-159f1be7fb60.png) да, если это произошло, вы должны изменить учетную запись.

Isn't it easier to install it on a computer?

**Comment by slooi** [comment]
Yes, you could just install and run this on your own local computer. However, the ImageAISetup.ipynb file downloads from third party servers which is a bit of a security concern. If you install and run this on your own machine, you could potentially compromise your system.
